app/mobile/src/package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "main": "node_modules/expo/AppEntry.js",
+  "scripts": {
+    "start": "expo start",
+    "android": "expo start --android",
+    "ios": "expo start --ios",
+    "web": "expo start --web",
+    "eject": "expo eject",
+    "storybook": "start-storybook -p 7007",
+    "test": "jest",
+    "build-storybook": "build-storybook"
+  },
+  "dependencies": {
+    "@react-native-async-storage/async-storage": "~1.15.0",
+    "expo": "~42.0.1",
+    "expo-cli": "^4.11.0",
+    "expo-status-bar": "~1.0.4",
+    "prettier": "^2.4.0",
```

Comment:
Move this to `devDependencies` as `prettier` is not needed for the final app to function